### PR TITLE
Added PGD ping file along with Slack integration id for PGD channel

### PR DESCRIPTION
**Pull request by that-guy-is-it**

### CODE REVIEW

File: pingdom-checks/patient-generated-data.ping [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# -*- sh -*-
+
+PGD_SLACK_ID=PLACEHOLDER
```
Comment (bschofield-va) [inline pingdom-checks/patient-generated-data.ping:3]: what's this?

File: pingdom-checks/patient-generated-data.ping [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+pingdom save-check \
+  --template fhir-resource \ 
+  -a name=sandbox-pgd-r4-patient \ 
+  -a host=sandbox-api.va.gov \
+  -a url="/services/fhir/pgd/v0/r4/questionnaire/37953b72-961b-41ee-bd05-86c62bacc46b" \
+  -a authorization_token= "$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
```
Comment (bschofield-va) [inline pingdom-checks/patient-generated-data.ping:10]: space between template argument and value `<space>=`

File: pingdom-checks/patient-generated-data.ping [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  -a userids_csv= "$NO_USERS" \
+  -a integrationids_csv= "$SLACK_PGD_SANDBOX" \
```
Comment (bschofield-va) [inline pingdom-checks/patient-generated-data.ping:12]: Remove the trailing line continuation that continues to nothing

**Review by bschofield-va — CHANGES_REQUESTED** [review]
see comments

File: pingdom-checks/patient-generated-data.ping (hunk 1 above)
Comment (that-guy-is-it) [reply pingdom-checks/patient-generated-data.ping:3]: Oh crap, that is something I put as a reminder to ask a question, and after I asked it I forgot to remove it. Well, that is embarrassing.

File: integration-ids.conf [hunk 4]
```diff
@@ -20,6 +20,7 @@ export SLACK_API_ALARMS_INFRASTRUCTURE=107273
 export SLACK_API_ALARMS_VA_FORMS=110162
 export SLACK_LIGHTHOUSE_ALERTS=104825
 export SLACK_VASDVP_MONITORING=86071
+export SLACK_PGD_SANDBOX=110372
```
Comment (jblefkove-va) [inline integration-ids.conf:23]: this should be `SLACK_API_ALARMS_XXX`

File: integration-ids.conf [hunk 5]
```diff
@@ -20,6 +20,7 @@ export SLACK_API_ALARMS_INFRASTRUCTURE=107273
 export SLACK_API_ALARMS_VA_FORMS=110162
 export SLACK_LIGHTHOUSE_ALERTS=104825
 export SLACK_VASDVP_MONITORING=86071
+export SLACK_API_ALARMS_PGD_SANDBOX=110372
```
Comment (jblefkove-va) [inline integration-ids.conf:23]: alphabetize

File: pingdom-checks/patient-generated-data.ping [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# -*- sh -*-
+
+pingdom save-check \
+  --template fhir-resource \ 
+  -a name=sandbox-pgd-r4-patient \ 
+  -a host=sandbox-api.va.gov \
+  -a url="/services/fhir/pgd/v0/r4/questionnaire/37953b72-961b-41ee-bd05-86c62bacc46b" \
+  -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
```
Comment (jblefkove-va) [inline pingdom-checks/patient-generated-data.ping:8]: our static token is different from the health apis static token

File: pingdom-checks/patient-generated-data.ping [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# -*- sh -*-
+
+pingdom save-check \
+  --template fhir-resource \ 
+  -a name=sandbox-pgd-r4-patient \
```
Comment (jblefkove-va) [inline pingdom-checks/patient-generated-data.ping:5]: pgd-r4-questionnaire

File: pingdom-checks/patient-generated-data.ping [hunk 8]
```diff
@@ -0,0 +1,11 @@
+# -*- sh -*-
+
+pingdom save-check \
+  --template fhir-resource \ 
+  -a name=sandbox-pgd-r4-questionnaire \ 
+  -a host=sandbox-api.va.gov \
+  -a url="/services/fhir/pgd/v0/r4/questionnaire/37953b72-961b-41ee-bd05-86c62bacc46b" \
+  -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
+  -a userids_csv="$NO_USERS" \
+  -a integrationids_csv="$SLACK_API_ALARMS_PGD_SANDBOX"
```
Comment (jblefkove-va) [inline pingdom-checks/patient-generated-data.ping:10]: this should also go to `SLACK_VASDVP_MONITORING`

File: integration-ids.conf [hunk 9]
```diff
@@ -17,6 +17,7 @@ export SLACK_API_ALARMS_DATA_QUERY=106657
 export SLACK_API_ALARMS_FACILITIES=107438
 export SLACK_API_ALARMS_FACILITIES_COLLECTOR=108531
 export SLACK_API_ALARMS_INFRASTRUCTURE=107273
+export SLACK_API_ALARMS_PGD_SANDBOX=110372
```
Comment (jblefkove-va) [inline integration-ids.conf:20]: `SLACK_API_ALARMS_PGD` is sufficient here. The channel will also receive production alerts when we add them later.

**Comment by jblefkove-va** [comment]
FYI @department-of-veterans-affairs/lighthouse-leeroyjenkles this adds monitoring for PGD sandbox. Since the app is not publicly advertised, it is not included on sandbox. Alerts will be sent to the `#lighthouse_alerts` and `#api-alarms-patient-generated-data` slack channels.

File: pingdom-checks/patient-generated-data.ping [hunk 10]
```diff
@@ -0,0 +1,11 @@
+# -*- sh -*-
+
+pingdom save-check \
+  --template fhir-resource \ 
+  -a name=sandbox-pgd-r4-questionnaire \ 
+  -a host=sandbox-api.va.gov \
+  -a url="/services/fhir/pgd/v0/r4/questionnaire/37953b72-961b-41ee-bd05-86c62bacc46b" \
+  -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
+  -a userids_csv="$NO_USERS" \
+  -a integrationids_csv="$SLACK_API_ALARMS_PGD,$SLACK_VASDVP_MONITORING"
```
Comment (jblefkove-va) [inline pingdom-checks/patient-generated-data.ping:10]: pull this list out into a separate var, like: https://github.com/department-of-veterans-affairs/lighthouse-external-monitoring/blob/master/pingdom-checks/data-query.ping#L3

**Comment by jblefkove-va** [comment]
@bschofield-va is this good to go?

**Review by bschofield-va — APPROVED** [review]

**Pull request merged**